IS0501Test.py

```diff
@@ -269,6 +269,8 @@ def test_11(self):
                       'rtcp_source_port']
         combinedParams = rtcpParams + fecParams
         rtcpParams = rtcpParams + generalParams
+        websocketParams = ['connection_uri', 'ext_is_07_rest_api_url', 'ext_is_07_source_id']

```

Comment:
The spec allows any parameter which begins 'ext_', so I'd suggest rather than including the explicit IS-07 ones here, any 'ext_' parameters should be stripped from the GET response before sorting and comparing it.